bccsp/pkcs11/pkcs11.go

```diff
@@ -129,7 +119,10 @@ func (csp *impl) returnSession(session pkcs11.SessionHandle) {
 // Look for an EC key by SKI, stored in CKA_ID
 func (csp *impl) getECKey(ski []byte) (pubKey *ecdsa.PublicKey, isPriv bool, err error) {
 	p11lib := csp.ctx
-	session := csp.getSession()
+	session, err := csp.getSession()
```

Comment:
Do the callers of this function (and the ones below) need to care about getSession erroring and returning early? Or does nothing change in the caller's logic?

Reply:
My general assumption was, because there already existed error paths in these functions, there was proper error handling in the caller code. The fact is, if the session is bad, the subsequent calls would fail. You never had to worry about that happening before because if the session failed, the peer process exited.

Probably worth me checking into the callers and make sure things are handled properly up the stack

Reply:
Yeah that's what I figured so my comment was more of a consideration. Otherwise LGTM

Reply:
It's a good consideration though, since we don't have an integration test on this code path, its difficult to say all of these pieces function perfectly well together